paddle/framework/lod_tensor.cc

```diff
@@ -286,62 +286,86 @@ void DeserializeFromStream(std::istream &is, LoDTensor *tensor,
   DeserializeFromStream(is, static_cast<Tensor *>(tensor), dev_ctx);
 }
 
-// TODO(tonyyang-svail): make this function support LoD
 std::vector<LoDTensor> LoDTensor::SplitLoDTensor(
```

Comment:
Seems `split_op` can do the same work. I use a transpiler to append `split_op` before send them in distributed training. I still worries that current "spliting" operations still need to copy the actual data, which can be optimized.